src/pages/admin/CustomerInstitutionAdminsForm.tsx

```diff
@@ -10,6 +10,7 @@ import { filterUsersByRole } from '../../utils/role-helpers';
 import UserList from './UserList';
 import Modal from '../../components/Modal';
 import { AddRoleModalContent } from './AddRoleModalContent';
+import { UserListSkeleton } from './UsersListSkeleton';
```

Comment:
irriterer meg at det heter `UsersListSkeleton`. Mulig å kalle fila `UserListSkeleton`? 😝 

```suggestion
import { UserListSkeleton } from './UserListSkeleton';
```